docs/pipeline.md

```diff
@@ -1,247 +1,165 @@
+ - Finally, the post-processing step takes care of harmonising the tool-specific outputs into standardised PhEval TSV format, ensuring uniformity and ease of analysis of results from all VGPAs. In this context, the tool-specific output is condensed to provide only two essential elements, the entity of interest, which can either be a variant, gene, or disease, and its corresponding score. PhEval then assumes the responsibility of subsequent standardisation processes. This involves the reranking of the results in a uniform manner, ensuring that fair and comprehensive comparisons can be made between tools.
 
-📦resources  
- ┣ 📜**Makefile.j2**  
+### 3. Analysis Phase
 
+In the analysis phase, PhEval generates comprehensive statistical reports based on
+standardised outputs from the runner phase.
 
-*custom.Makefile* is the template that will be generated on the fly based on the *pheval-config.yaml*. Each of these configurations is filled using a syntax like this: ```{{ config.tool }}```. The value between the curly brackets is replaced by the corresponding configuration in the configuration file.
+## Customising PhEval Pipeline Experiments 
 
----
+The PhEval pipeline is orchestrated using a Makefile Jinja template strategy. Therefore, to describe a new experiment in the pipeline, the user needs to generate a Makefile workflow based on a configuration file.
 
-## PhEval custom.Makefile
+In the resources folder are the following files responsible for Makefile generation:
 
 📦resources  
- ┣ 📜**custom.Makefile**  
-
----
-## PhEval generatemakefile.sh
+┣ 📜Makefile.j2  
+┣ 📜custom.Makefile  
+┣ 📜generatemakefile.sh  
+┗ 📜pheval-config.yaml  
 
-📦resources  
- ┣ **📜generatemakefile.sh**  
+Let's begin by describing the `pheval-config.yaml` file and its structure.
 
+### PhEval Configuration File
 
-*generatemakefile.sh* is only a shortcut for Makefile rendering using the configuration file e.g.
+This file is responsible define the experiment settings and will be used to generate the Makefile using a Jinja template which consumes this YAML configuration file.
 
-    bash ./resources/generatemakefile.sh
+#### Directories Section
 
-## PhEval Configuration File
+The `data` and `tmp` properties are mandatory and must be specified in this section.
 
-In resources folder, there is a file named *pheval-config.yaml*, this file is responsible for storing the PhEval Makefile generation.
+- `data` property refers to the folder location where the necessary phenotypic data for the pipeline will be downloaded and extracted.
```

Comment:
Is the `data` property specific to Exomiser? What will happen if it is left blank and users don't want to run Exomiser?